react/features/filmstrip/functions.web.js

```diff
@@ -170,7 +171,8 @@ export function calculateThumbnailSizeForVerticalView(clientWidth: number = 0) {
 
     return {
         local: {
-            height: Math.floor(availableWidth / interfaceConfig.LOCAL_THUMBNAIL_RATIO),
+            height: Math.floor(availableWidth
+                / (interfaceConfig.LOCAL_THUMBNAIL_RATIO / DEFAULT_LOCAL_TILE_ASPECT_RATIO)),
```

Comment:
Why do we divide by `DEFAULT_LOCAL_TILE_ASPECT_RATIO`? 

Did you meant `||` instead of '/'? If this is the case  this function becomes identical to `calculateThumbnailSizeForResizableVerticalView`, isn't it?

Reply:
Yeah, I meant `||` . They're not identical, but I'll merge them into one function